fgpyo/util/metric.py

```
@@ -221,6 +223,8 @@ def read(cls, path: Path, ignore_extra_fields: bool = True) -> Iterator[Any]:
         Args:
             path: the path to the metrics file.
             ignore_extra_fields: True to ignore any extra columns, False to raise an exception.
+            strip_whitespace: True to strip leading and trailing whitespace from each field, 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)